### PR TITLE
DEV-551/hide-crash-menu-behind-environment-variable

### PR DESCRIPTION
**Pull request by MiladNazeri**
https://highfidelity.atlassian.net/browse/DEV-551

### CODE REVIEW

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -725,45 +725,51 @@ Menu::Menu() {
         DependencyManager::get<PickManager>().data(), SLOT(setForceCoarsePicking(bool)));
 
     // Developer > Crash >>>
-    MenuWrapper* crashMenu = developerMenu->addMenu("Crash");
+    bool result = false;
+    const QString HIFI_SHOW_SOURCE_DEVELOPER_CRASH_MENU("HIFI_SHOW_SOURCE_DEVELOPER_CRASH_MENU");
```
Comment (zfox23) [inline interface/src/Menu.cpp]: why is "source" a part of this env var and variable name?

Comment (MiladNazeri) [reply interface/src/Menu.cpp]: oops sorry!  Fixing

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21254/HighFidelity-Beta-PR16175-f9c5d00.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21254/HighFidelity-Quest-Beta-PR16175-f9c5d00.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21254/HighFidelity-Beta-PR16175-f9c5d00.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21254/HighFidelity-Beta-Interface-PR16175-f9c5d00.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21254/HighFidelity-Beta-PR16175-f9c5d00.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21254/HighFidelity-Beta-Interface-PR16175-f9c5d00.exe

**Pull request merged**